torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -4110,7 +4129,7 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::allreduce_sparse(
   TORCH_CHECK(tensors.size() == 1, MULTI_DEVICE_ERROR_MSG);
   auto tensor = tensors.back();
   TORCH_CHECK(
-      !isFloat8Type(tensor.scalar_type()),
+      !isUnsupportedFloat8(tensor.scalar_type()),
       "Float8 dtypes are not currenlty supported for NCCL reductions");
```

Comment:
should we changes these comments to indicate which types are supported?

applies below as well

Reply:
Good point.

Reply:
Also is isFloat8Type still being used? If not, maybe you want to consider to remove it as well?